classes/Hook.php

```diff
@@ -89,19 +91,46 @@ public function add($autodate = true, $null_values = false)
         return parent::add($autodate, $null_values);
     }
 
+    public static function isDisplayHookName($hook_name)
```

Comment:
This won't work for "retro" hook names. 
I'm doing the same thing in my branch, maybe you could be interested by the `normalizeHookName` function? https://github.com/alexsegura/PrestaShop/commit/7de4d37be76d7386f96b9503bfde75c24f87dccb